<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="Sidebar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(31, 31, 31);</string>
   </property>
   <widget class="QLabel" name="l_question">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1200</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 24pt;
    font-weight: 1200;
    color: #919191;
    text-align: center;
}
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#919191;&quot;&gt;&amp;lt; Question Here &amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

border-radius:25px;
background-color: rgb(148, 148, 148);

}

QPushButton::hover {

	background-color: rgb(213, 213, 213);

}

QPushButton::pressed {

	background-color: rgb(148, 148, 148);

}
</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_help">
    <property name="geometry">
     <rect>
      <x>1139</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

border-radius:25px;
background-color: rgb(148, 148, 148);

}

QPushButton::hover {

	background-color: rgb(213, 213, 213);

}

QPushButton::pressed {

	background-color: rgb(148, 148, 148);

}
</string>
    </property>
    <property name="text">
     <string>!?</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_help">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;color: rgb(170, 0, 255);
text-align: right</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&amp;lt;Help&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Mainbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>1200</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(57, 57, 57);</string>
   </property>
   <widget class="QPushButton" name="btn_01">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>300</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

border-radius:20px;
background-color: rgb(148, 148, 148);

font: 15pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton::hover {

	background-color: rgb(213, 213, 213);

}

QPushButton::pressed {

	background-color: rgb(148, 148, 148);

}
</string>
    </property>
    <property name="text">
     <string>&lt;Option 01&gt;</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_02">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>300</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

border-radius:20px;
background-color: rgb(148, 148, 148);

font: 15pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton::hover {

	background-color: rgb(213, 213, 213);

}

QPushButton::pressed {

	background-color: rgb(148, 148, 148);

}
</string>
    </property>
    <property name="text">
     <string>&lt;Option 02&gt;</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_04">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>300</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

border-radius:20px;
background-color: rgb(148, 148, 148);

font: 15pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton::hover {

	background-color: rgb(213, 213, 213);

}

QPushButton::pressed {

	background-color: rgb(148, 148, 148);

}
</string>
    </property>
    <property name="text">
     <string>&lt;Option 04&gt;</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_03">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>300</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

border-radius:20px;
background-color: rgb(148, 148, 148);

font: 15pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton::hover {

	background-color: rgb(213, 213, 213);

}

QPushButton::pressed {

	background-color: rgb(148, 148, 148);

}
</string>
    </property>
    <property name="text">
     <string>&lt;Option 03&gt;</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
